data/filters/youtube-cleanup.yaml

```
@@ -0,0 +1,49 @@
+title: Cleanup YouTube pages
+params:
+  - name: channel-clarification
+    description: Hide the channel description box under videos and shorts.
+    type: checkbox
+    default: false
+  - name: remove-text-after-buttons-below-video
+    description: Hide the text next to the dislike/share/download/report/save buttons below a video.
+    type: checkbox
+    default: false
+tags:
+  - youtube
+template: |
+  {{#if channel-clarification}}
+  www.youtube.com###clarify-box
+  www.youtube.com##ytd-shorts .disclaimer-container:upward(#info-panel)
+  {{/if}}
+  {{#if remove-text-after-buttons-below-video}}
+  www.youtube.com###info > * [id^="top-level-buttons"] [class*="button-renderer"] [id="text"]:not(:has-text(/\d/))
+  www.youtube.com##.ytd-watch-metadata > .top-level-buttons #text:not(:has-text(/[\d]/))
+  {{/if}}
+tests:
+  - params: {}
+    output: ""
+  - params:
+      remove-text-after-buttons-below-video: true
+      channel-clarification: true
+    output: |
+      www.youtube.com###clarify-box
+      www.youtube.com##ytd-shorts .disclaimer-container:upward(#info-panel)
+      www.youtube.com###info > * [id^="top-level-buttons"] [class*="button-renderer"] [id="text"]:not(:has-text(/\d/))
+      www.youtube.com##.ytd-watch-metadata > .top-level-buttons #text:not(:has-text(/[\d]/))
+  - params:
+      remove-text-after-buttons-below-video: true
+     output: |
```

There is an extra space here that breaks the YAML parsing. Tests should be green with this change.
```suggestion
    output: |
```